### DISCUSSION TITLE
Add Goal Conversions section on Embed dashboard

### DISCUSSION THREAD

**Discussion by gcasasi** [plausible/analytics#2383] [Anything Else]
Hi, when I use Embed dashboard of my sites, this embed doesn't contain Goal Conversions section, it's possible to enable it? 

Thanks

**Comment by bogplau** [comment]
Hi there, 

Can you please[ send us an email](https://plausible.io/contact) with your issue? I need some more information that shouldn't be public, in order to troubleshoot this.

Thank you!

NOTE: Once this is resolved, I will post the resolution here in case anybody faces the same issue.

**Reply by gcasasi** [reply]
Hi, I'm Sorry, now I know what happens, I only need to change height of (iframe) to >2000px.

**Reply by bogplau** [reply]
Thanks for this, G.!
